std/algorithm/package.d

```diff
@@ -168,8 +168,6 @@ sort!"a > b"(a);           // predicate as string
 sort(a);                   // no predicate, "a < b" is implicit
 ----
 
-Macros:
```

Comment:
Must not remove this instance of `Macros:` as there are other macros being defined here. Happened a bunch of times throughout the diff.